### ISSUE TITLE
Binary files in repo

### ISSUE THREAD

**Issue by Lignum** [hugeblank/Sonar-X#1]
Hi, just wanted to point out that you have binary files in your repo. [This article](https://development.robinwinslow.uk/2013/06/11/dont-ever-commit-binary-files-to-git/) is a great resource on why this is bad and what to do about it. Since there's only 4 commits, it might be a smart idea to just re-create the repo.

**Comment by hugeblank** [comment]
@Lignum yeah I plan on deleting the repo entirely, as I have been notified that I am infringing on multiple 
 mod's terms of use. What I think I will do is make a script that just "wget"'s all the mods.
